disnake/http.py

```diff
@@ -206,7 +207,7 @@ def __init__(
         self.loop: asyncio.AbstractEventLoop = asyncio.get_event_loop() if loop is None else loop
         self.connector = connector
         self.__session: aiohttp.ClientSession = MISSING  # filled in static_login
-        self._locks: weakref.WeakValueDictionary = weakref.WeakValueDictionary()
+        self._locks: weakref.WeakValueDictionary = weakref.WeakValueDictionary()  # type: ignore
```

Comment:
```suggestion
        self._locks: weakref.WeakValueDictionary[str, asyncio.Lock] = weakref.WeakValueDictionary()
```

Reply:
Oh, thanks, I should have left that untouched as I didn't know how to type-hint that